bin-src/lib/uploadZip.js

```diff
@@ -60,6 +60,10 @@ export async function waitForUnpack(ctx, url) {
       try {
         res = await ctx.http.fetch(url, {}, { retries: 0, noLogErrorBody: true });
       } catch (e) {
+        const { response = {} } = e;
+        if (response.status === 403) {
```

Comment:
Apparently webpack hates optional chaining here and throws errors in our tests. 😭